src/barnetilsyn/steg/5-aktivitet/AktivitetConfig.tsx

```diff
@@ -48,17 +48,11 @@ export const hvaErDinArbeidssituasjonSpm = (intl: IntlShape): ISpørsmål => ({
   flersvar: true,
   svaralternativer: [
     {
-      id: EAktivitet.erArbeidstaker,
+      id: EAktivitet.erArbeidstakerOgEllerLønnsmottakerFrilanser,
```

Comment:
Håndterer vi den nye svarid-en i ef-sak eller blir dette breaking? 🤔 
(Vet ikke om vi har tatt den ibruk ennå, forsåvidt)

Reply:
Det går faktisk fint! 

Vi bruker egentlig ikke svariden videre til no annet enn å få riktig "tittel". Har nå oppdatert funksjonen som bruker svarid til å hente overskrift nå med ny svarid.
For å sjekke om bruker har valgt "arbeidstaker/frilanser" så sjekker vi bare om `Aktivitiet.arbeidsforhold` eksisterer. Hvis ja, så viser vi dataen. :) 

Har testet, og det skal gå fint.

Reply:
<img width="684" alt="Skjermbilde 2021-03-18 kl  16 44 33" src="https://user-images.githubusercontent.com/8249453/111655055-9b81f280-8809-11eb-9b36-ddacd3fd886e.png">

i ef-sak bruker vi ikke enumer for svaridene. `Aktivitet.hvaErDinArbeidssituasjon` er en liste med strenger. Så de bryr seg ikke om vi endrer på enumen i frontend :)